charts/drupal/templates/drupal-configmap.yaml

```diff
@@ -40,6 +40,7 @@ data:
     request_order = GP
     post_max_size = {{ .Values.php.php_ini.post_max_size }}
     upload_max_filesize = {{ .Values.php.php_ini.upload_max_filesize }}
+    max_execution_time = {{ .Values.php.php_ini.max_execution_time }}
```

Comment:
max_execution_time is already set on line 34. Please replace that line with this new one.

Reply:
Fixed this myself